script/events/room.js

```diff
@@ -645,6 +651,43 @@ Events.Room = [
 					}
 				}
 			},
+			'killman':{
+				text: [
+					_("beaten to a pulp"),
+					_("the man had very little"),
+					_("you see his family in the distance"),
+					_("was it worth it")
+				],
+				buttons: {
+					'bye': {
+						text: _('shamefully gather goods'),
+						nextScene: 'end'
+					}
+			},
+			'killmanlots':{
+				text: [
+					_("a life brutally ended"),
+					_("the man carried lots"),
+					_("your rewarded greatly")
+				],
+				buttons: {
+					'bye': {
+						text: _('gather goods'),
+						nextScene: 'end'
+					}
+			},
+			'killmannothing':{
+				text: [
+					_("the man lays there."),
+					_('he had nothing to give'),
+					_('is it really worth it')
```

Comment:
I'd remove this line. Don't tell the player how to feel; set the scene and *make* them feel.